### ISSUE TITLE
Accessibility: community building committee

### ISSUE THREAD

**Issue by EbubeEvan** [jupyterhub/outreachy#67]
### What page is this for?

https://jupyter.org/governance/communitybuildingcommittee.html

### WAVE accessibility report

I worked on https://jupyter.org/governance/communitybuildingcommittee.html and I found the following issues:

**Errors**
- Language missing or invalid: a lang="en" should be added to the open `html` tag to specify the language of the webpage in order to increase its accessibilty.
- Empty form labels : the `label` elements should be deleted and the `i` elements nested within them should be nested within `span` elements. Aria-label attributes should then be added to the `span` elements to describe the function of the `i` elements.
- Multiple form labels: since the `input` tag contains an aria-label, one of the `label` elements should be deleted. A form control can only have one element.
- Empty button: text isn't necessary for the full screen view button because its icon is self explanatory. But for the sake of the screen reader, a value attribute should be added to the `button` to describe its function.

**Features**
- Linked image with alternative text: the alt attribute doesn't quite describe the image effectively. It uses an abbreviation which a visually impaired user might not understand. The description should be fully written out.

**Structural Elements**
- 3 X Heading level 1: There can only be one `h1` element in a web page. The other headings that are not the main heading should be changed to `h2`.
- 4 X Heading level 2: 2 of them are improperly placed. They should be replaced by `h3` headings.

**ARIA**
- 5X ARIA: the ARIA roles should be deleted and the parent elements replaced with `h3`.

![Screenshot 2022-10-12 181128](https://user-images.githubusercontent.com/102484584/195406083-af36640b-448c-4f76-ba0b-78e3bcfcf6d9.png)

### The HTML element to be changed

```html
//Errors
<html native-dark-active="" style="margin-left: 380px;">
<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>
<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>
<label for="toctree-checkbox-3">
<i class="fas fa-chevron-down"></i>
</label>
<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>
<label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
Form label
<label class="overlay overlay-navbar" for="__navigation">
<div class="visually-hidden">
Toggle navigation sidebar
</div>
</label>
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
Form label
<label class="overlay overlay-pagetoc" for="__page-toc">
<div class="visually-hidden">
Toggle in-page Table of Contents
</div>
</label>
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>

//Features
<a rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/">
Linked image with alternative text
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">
</a>
<br>

//Structural Elements
<h1 class="site-logo" id="site-title">
Project Jupyter Governance
</h1>
<h2>
Reporting
<a class="headerlink" href="#reporting" title="Permalink to this headline">
#
</a>
</h2>
<h2>
Bootstrapping
<a class="headerlink" href="#bootstrapping" title="Permalink to this headline">
#
</a>
</h2>

//ARIA
<p aria-level="2" class="caption" role="heading">
<span class="caption-text">
Code of Conduct
</span>
</p>
<p aria-level="2" class="caption" role="heading">
<span class="caption-text">
Governance
</span>
</p>
<p aria-level="2" class="caption" role="heading">
<span class="caption-text">
Organizational Structure
</span>
</p>
<p aria-level="2" class="caption" role="heading">
<span class="caption-text">
Organizational Policy
</span>
</p>
<p aria-level="2" class="caption" role="heading">
<span class="caption-text">
Future Governance [work in progress]
</span>
</p>
```

### Your proposed HTML (or CSS), after change

```html
//Errors
<html lang="en native-dark-active="" style="margin-left: 380px;">
<span aria-label="toggle">
<i class="fas fa-chevron-down"></i>
</span>
<span aria-label="toggle">
<i class="fas fa-chevron-down"></i>
</span>
<span aria-label="toggle">
<i class="fas fa-chevron-down"></i>
</span>
<span class="headerbtn__icon-container" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<i class="fas fa-bars"></i>
</span>
<span class="headerbtn__icon-container headerbtn headerbtn-page-toc">
<i class="fas fa-list"></i>
</span>
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
Form label
<label class="overlay overlay-navbar" for="__navigation">
<div class="visually-hidden">
Toggle navigation sidebar
</div>
</label>
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode" value="fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>

//Features
<a rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/">
Linked image with alternative text
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="Creative Commons domain">
</a>
<br>

//Structural Elements
<h2 class="site-logo" id="site-title">
Project Jupyter Governance
</h2>
<h2>
Community Building Committee
</h2>
<h3>
Reporting
<a class="headerlink" href="#reporting" title="Permalink to this headline">
#
</a>
</h3>
<h3>
Bootstrapping
<a class="headerlink" href="#bootstrapping" title="Permalink to this headline">
#
</a>
</h3>

//ARIA
<h3 class="caption caption-text">
Code of Conduct
</h3>
<h3 class="caption caption-text">
Governance
</h3>
<h3 class="caption caption-text">
Organizational Structure
</h3>
<h3 class="caption caption-text">
Organizational Policy
</h3>
<h3 class="caption caption-text" role="heading">
Future Governance [work in progress]
</h3>
```

### More info

The aria-label="toggle" in the accordion icon can be controlled dynamically by javascript to change to toggle-up when closed and toggle-down when open.

**Comment by minrk** [comment]
Thanks for these suggestions!

**Comment by EbubeEvan** [comment]
It was my pleasure
I learned a lot from this project

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
